webcompat/views.py

```diff
@@ -73,10 +73,15 @@ def format_date(datestring):
 @app.route('/login')
 def login():
     if session.get('user_id', None) is None:
-        # manually set the referer so we know where to come back to
-        # when we return from GitHub
-        set_referer(request)
-        return github.authorize('public_repo')
+        if app.config['TESTING']:
+            session['username'] = 'testuser'
+            session['avatar_url'] = '/img/avatar.png?'
```

Comment:
Yeah, it was intentional, but anyways it's bet to put it in the fixtures folder I'll change that.

I put it there because for some reason there was a string being added to the end of the URL and it was returning a 404.